lib/Translation/ExportVerilog/ExportVerilog.cpp

```diff
@@ -883,6 +885,15 @@ SubExprInfo ExprEmitter::visitComb(ConcatOp op) {
   return {Unary, IsUnsigned};
 }
 
+SubExprInfo ExprEmitter::visitComb(BitcastOp op) {
+  Type toType = op.getType();
+  os << "/*cast(bit";
+  emitTypeDims(toType, op.getLoc(), os);
+  os << ")*/";
+  auto subPrec = emitSubExpr(op.from(), LowestPrecedence);
+  return {Unary, subPrec.signedness};
```

Comment:
Returning unary here isn't correct, you should return `subPrec` as a whole, ala `return emitSubExpr(...`